package.json

```diff
@@ -7,11 +7,9 @@
   "main": "./index.js",
   "types": "./index.d.ts",
   "bin": {
-    "node-core-test": "./bin/test_runner.js"
+    "node--test": "./bin/node--test.js"
```

Comment:
I'm not sure about this change, it's more of a shower thought: if we call the binary `node--test`, folks would simply have to add a space to use the built-in (`node--test` -> `node --test`). wdyt?

Reply:
Interesting! I think in general it's nice etiquette if an npm package's binary is called the exact same name as the package, for collisions and to keep your environment clean, but here it's a special case. While `node--test` looks weird to me, on the upside there's no npm package called `node--test` currently, so collisions aren't likely, and `node --test` is the definitive name that it mirrors and probably won't ever change. I think I'm +1 for it, but would suggest to also keep the old binary name.

Reply:
this will require a docs update too

Reply:
That’s a good point, we don’t have to restrict ourselves to only one binary 👍